### ISSUE TITLE
Toto-2.0 should be "zero-shot" instead of "pre-trained"?

### ISSUE THREAD

**Issue by siriuz42** [SalesforceAIResearch/gift-eval#134]
Hi Gerald @gorold,

Toto-2.0s are pretty awesome. Glad to see a new family of good models on GIFT.

Wanted to ask a quick clarification question: 

Based on [Toto-2.0 tech report](https://www.datadoghq.com/blog/ai/toto-2/) these models are not trained using data in the domains gift-eval covers. In this regard should these models be marked as "zero-shot" instead of "pre-train" under which the models would presumably touch the train split of gift-eval during pretraining?

**Comment by gorold** [comment]
Hey Yichen, we just released a more detailed [tech report](https://arxiv.org/abs/2605.20119v1), which describes our hyperparameter tuning pipeline, and importantly, we did use the GIFT-Eval validation split for hyperparameter tuning, which ultimately gives it a "pre-train" label rather than "zero-shot" label.
